client/src/components/Register/Introduction.scss

```diff
@@ -0,0 +1,53 @@
+$feature-list-content-margin: 20px;
+$feature-list-icon-size: 50px;
+$feature-list-action-list-spacing: 45px;
+
+$font-feature-list-title-size: 16px;
+$font-feature-list-title-weight: bold;
```

Comment:
Let's put these in the mfa-variables sheet

Reply:
Given that they only relate to the styles in this sheet, it made more sense to me to co-locate them. Having one giant sheet with nothing but variables for every section of the styles feels messy to me. There are definitely variables that belong in the global scope (colours, base typography, etc.), but to me, styles for a specific component don't.